### PR TITLE
Added Book entity, Book service and Book repository.

### PR DESCRIPTION
**Pull request by sovisou**

### CODE REVIEW

File: src/main/java/com/onlinebookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/model/Book.java]:
```suggestion
@Getter
@Setter
```

Don't use Data for entities

File: src/main/java/com/onlinebookstore/repository/impl/BookRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import com.onlinebookstore.repository.BookRepository;
+import java.util.List;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/repository/impl/BookRepositoryImpl.java:14]: use Lombok annotation to create constructor

File: src/main/java/com/onlinebookstore/repository/impl/BookRepositoryImpl.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        return book;
+    }
+
+    @Override
+    public List<Book> findAll() {
+        List<Book> books;
+        try (Session session = sessionFactory.openSession()) {
+            books = session.createQuery("FROM Book", Book.class).getResultList();
+            return books;
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/repository/impl/BookRepositoryImpl.java]:
```suggestion
        try (Session session = sessionFactory.openSession()) {
            return session.createQuery("FROM Book", Book.class).getResultList();
```

File: src/main/java/com/onlinebookstore/service/impl/BookServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.onlinebookstore.service.impl;
+
+import com.onlinebookstore.model.Book;
+import com.onlinebookstore.repository.BookRepository;
+import com.onlinebookstore.service.BookService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/service/impl/BookServiceImpl.java:10]: use Lombok annotation to create constructor

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve it

**Review by okuzan — APPROVED** [review]

**Pull request merged**